code-challenges.test.js

```diff
@@ -11,42 +11,174 @@
+console.log(filterLetter(fruitArray(letterE)))
+
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
+// psuedocode:
+// create function named bigWin that takes in a array and outputs an "string" function must look at each number to see if the given numbers how many duplicate numbers there are. must be a counter to track numbers thinking i may he to do if statements
+// input: array and "string"
+// output:"string"
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
+describe("bigWin", () => {
+  it("takes in an array of 5 numbers and determines whether or not the array is a full house.", () => {
+    const hand1 = [5, 5, 5, 3, 3];
+    // Expected output: true
+    const hand2 = [5, 5, 3, 3, 4];
+    // Expected output: false
+    const hand3 = [5, 5, 5, 5, 4];
+    // Expected output: false
+    const hand4 = [7, 2, 7, 2, 7];
+    // Expected output: true
+
+    expect(bigWin(hand1)).toEqual(["true"]);
+    expect(bigWin(hand2)).toEqual(["false"]);
+    expect(bigWin(hand3)).toEqual(["false"]);
+    expect(bigWin(hand4)).toEqual(["true"]);
+  });
+});
+
+// ---> FAIL  ./code-challenges.test.js
+// bigWin
+// ✕ takes in an array of 5 numbers and determines whether or not the array is a full house. (1 ms)
+// ---> ReferenceError: bigWin is not defined
+
+const hand1 = [5, 5, 5, 3, 3];
 // Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
+const hand2 = [5, 5, 3, 3, 4];
 // Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
+const hand3 = [5, 5, 5, 5, 4];
 // Expected output: false
-const hand4 = [7, 2, 7, 2, 7]
+const hand4 = [7, 2, 7, 2, 7];
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+// Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind. 
+
+// const arr1 = [0, 1, 2, [3, 4]]; (found on mdn)
+// console.log(arr1.flat()) ---> Expected output: Array [0, 1, 2, 3, 4]
+
+// const pets = ['cat', 'dog', 'bat']; (found on mdn)
+// console.log(pets.includes('cat'));
+// Expected output: true
+
+const bigWin = (array) => {
+    for (let i = 0; i < array.length; i++)
+}
+// I am going to be honest I tried and I looked up how to do this and i found a way to do it but I had no idea of what was going on in the function so I did not use it. This challenge stumped me way more than the others. I knew I had to iterate over the array twice but i didnt know how to accomplish that without using my index twice and having the index check against itself. I don't know if that is even possible. I have no clue how I even came to that point but I just have this crazy feeling that that's possible and it'll confuse the program. I just couldn't figure out how to word my search parameters to specify that.  
```

Comment:
Not sure what your thought process was here. The commented out code on lines 174-179 don't include any context or explanation, so I can't see how they are correlating to this challenge. I highly suggest you go over the fundamentals and basics covered during weeks 1 & 2 with your mentor until you have a firm understanding of those basic concepts.